docs/reporting.rst

```diff
@@ -0,0 +1,164 @@
+
+The reporting page for the tedana decomposition presents a series
+of interactive plots designed to help you evaluate the quality of your
+analyses. This page describes the plots forming the reports and well as
+infromation on how to take advantage of the interactive functionalities.
+
+Report Structure
+================
+
+The image below shows a representative report, which has two sections: a) the summary view,
+and b) the individual component view.
+
+.. image:: /_static/rep01_overallview.png
+  :align: center
+Summary View
+------------
+This view provides an overview of the decomposition and component
+selection results. It includes four different plots.
+
+* **Kappa/Rho Scatter:** This is a scatter plot of `Kappa` vs. `Rho` features for all components.
+  In the plot, each dot represents a different component. The x-axis represents the kappa feature, and the
+  y-axis represents the rho feature. These are two of the most
+  informative features describing the likelihood of the component
+  being BOLD or non-BOLD. Additional information is provided via color
+  and size. In particuale, color informs about its classification
+  status (e.g., accepted, rejected); while size relates to
+  the amount of variance explained by the component (larger dot,
+  larger variance).
+.. image:: /_static/rep01_kapparhoScatter.png
+  :align: center
+  :height: 400px
+
+* **Kappa Scree Plot:** This scree plots provides a view of the components ranked by `Kappa`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_kappaScree.png
+  :align: center
+  :height: 400px
+
+* **Rho Scree Plot:** This scree plots provides a view of the components ranked by `Rho`
+  As in the previous plot, each dot represents a component. The color of the dot inform us
+  about classification status. In this plot, size is not related to variance explained.
+.. image:: /_static/rep01_rhoScree.png
+  :align: center
+  :height: 400px
+
+* **Variance Explained Plot:** This pie plot provides a summary of how much variance is explained
+  by each individual component, as well as the total variance explained by each of the three
+  classification categories (i.e., accepted, rejected, ignored). In this plot, each component is
+  represented as a wedge, whose size is directly related to the amount of variance explained. The
+  color of the wedge inform us about the classification status of the component. For this view,
+  components are sorted by classification first, and inside each classification group by variance
+  explained.
+.. image:: /_static/rep01_varexpPie.png
+  :align: center
+  :height: 400px
+
+Individual Component View 
+-------------------------
+This view provides detailed information about an individual 
+component (selected in the summary view, see below). It includes three different
+interactive plots.
```

Comment:
These plots do not currently appear to be interactive. Also, the brain plot is not described here.

Reply:
That's why I brought the OHBM demo up. We could add the link to cover for the interactiveness.

Reply:
Sorry, I mean that, in the actual report, these figures are not interactive. They're still static. I didn't keep up with the report PR as much as I should have, but I assume that brainsprite was either dropped or pushed off to a future pull request.

Reply:
But yes I agree that linking to example reports would be very useful.

Reply:
Yes, the brainsprite stuff was pushed off to a future PR. Okay, if the other agree, I'll change my review to request changes.

Reply:
So should we just not list the plot as interactive?

Reply:
+1 on this route -- I've made suggestions to that effect in my review !